eng/scripts/vs.16.buildtools.json

```diff
@@ -16,6 +16,10 @@
         "Microsoft.VisualStudio.Component.VC.ATL.ARM64",
         "Microsoft.VisualStudio.Component.VC.Tools.ARM64",
         "Microsoft.VisualStudio.Component.VC.Tools.x86.x64",
+        "Microsoft.VisualStudio.Component.VC.14.29.16.11.ARM64",
+        "Microsoft.VisualStudio.Component.VC.14.29.16.11.x86.x64",
+        "Microsoft.VisualStudio.Component.VC.14.29.16.11.ATL.ARM64",
+        "Microsoft.VisualStudio.Component.VC.14.29.16.11.ATL",
```

Comment:
These components don't exist for VS 2019. The good news is the entries just above are already for v142.

Revert changes in all four vs.16.*.json files.

Reply:
Reverted